internal/voting/sql.go

```diff
@@ -125,6 +131,13 @@ const (
 	// alreadyProcessed checks if a resolution has already been processed
 	alreadyProcessed = `SELECT id FROM ` + votingSchemaName + `.processed WHERE id = $1;`
 
+	// manyProcessed checks if many resolutions have already been processed
+	// it will return a boolean for each resolution in the input array
+	manyProcessed = `WITH input AS (SELECT unnest($1::BYTEA[]) AS id)
```

Comment:
Same basic comment as the other query (can return a subset of processed ids rather than a bool set of same length and order), but nbd.